model/otlp/json_test.go

```diff
@@ -55,11 +55,11 @@ var logsOTLP = func() pdata.Logs {
 	il := rl.InstrumentationLibraryLogs().AppendEmpty()
 	il.InstrumentationLibrary().SetName("name")
 	il.InstrumentationLibrary().SetVersion("version")
-	il.LogRecords().AppendEmpty().SetName("testMetric")
+	il.LogRecords().AppendEmpty().SetSeverityText("testMetric")
```

Comment:
The usage here is a bit confusing to me as you're setting what used to be the name as a severity text, I would expect this field to be something like "INFO" or "WARN" rather than testMetric. I understand this is an arbitrary string field but it may cause confusion for others.

Reply:
Fixed.